boards/mega-xplained/Makefile.include

```diff
@@ -2,23 +2,12 @@
 export CPU = atmega1284p
 
 # configure the terminal program
-export PORT_LINUX  ?= /dev/ttyACM0
-export PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbserial*)))
-export BAUD        ?= 9600
-include $(RIOTMAKE)/tools/serial.inc.mk
-
-export FLASHER = avrdude
-export DIST_PATH = $(RIOTBOARD)/$(BOARD)/dist
-export DEBUGSERVER_PORT = 4242
-export DEBUGSERVER = $(DIST_PATH)/debug_srv.sh
-export DEBUGSERVER_FLAGS = "-g -j usb :$(DEBUGSERVER_PORT)"
-export DEBUGGER_FLAGS = "-x $(RIOTBOARD)/$(BOARD)/dist/gdb.conf $(ELFFILE)"
-export DEBUGGER = $(DIST_PATH)/debug.sh $(DEBUGSERVER_FLAGS) $(DIST_PATH) $(DEBUGSERVER_PORT)
+PORT_LINUX ?= /dev/ttyACM0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+PROGRAMMER_SPEED ?= 9600
 
 # PROGRAMMER defaults to the Bus Pirate ISP
-export PROGRAMMER ?= buspirate
-
-export PROGRAMMER_FLAGS = -P /dev/ttyUSB0
+PROGRAMMER ?= buspirate
+FFLAGS += -p m1284p
 
-OFLAGS += -j .text -j .data
-export FFLAGS += -p m1284p -c $(PROGRAMMER) $(PROGRAMMER_FLAGS) -F -U flash:w:$(HEXFILE)
+include $(RIOTBOARD)/common/arduino-atmega/Makefile.include
```

Comment:
Should probably be in common/atmega, since this isn't an Arduino. Do you think the Makefile.include should be moved from arduino-atmega to atmega?

Reply:
Maybe, have to take a look at it.

Reply:
> Do you think the Makefile.include should be moved from arduino-atmega to atmega?

I would say that some parts (FLASHER, DEBUGGER, etc) of `Makefile.include` in `common/arduino-atmega` should go to a `Makefile.include` in `common/atmega` and `common/arduino-atmega/Makefile.include` should include `common/atmega/Makefile.include`